src/www/widgets/widgets/picture.widget.php

```diff
@@ -87,9 +87,8 @@
 <?php
 if ($config['widgets']['picturewidget_filename'] != "") :?>
   <div id="picture-widgets" style="padding: 5px">
-    <img scr="/widgets/widgets/picture.widget.php?getpic=true" target="_blank">
-      <img style="border:0px solid; width:100%; height:100%" src="/widgets/widgets/picture.widget.php?getpic=true" alt="picture" />
-    </img>
+    <img scr="/widgets/widgets/picture.widget.php?getpic=true" target="_blank" style="border:0px solid; width:100%; height:100%" src="/widgets/widgets/picture.widget.php?getpic=true" alt="picture" />
+      </img>
```

Comment:
^^ already closed

Reply:
OK. I haven't tried it now. but i have now shortened the source code to an img tag. would that be okay?

Reply:
you can leave the img, but don't try to close it again. 

Please, pretty please, check and test code before pushing, I don't mind reviewing code from time to time, but a one-liner fix turns into a multitude of work on my end. Just try to keep it simple, a good starting point is to open an issue (for tracking reasons) explaining your goal(s) and an PR solving the issue. only a PR is also ok as long as it's very clear what's being changed, why it should change and what would be the downside of not doing it.

Reply:
yes it works perfect!